### DISCUSSION TITLE
[@types/jest] Why require dependencies? Need types for jest v27.

### DISCUSSION THREAD

**Discussion by blazkovicz** [DefinitelyTyped/DefinitelyTyped#53727] [Issues with a @types package]
1) Why does package require dependencies? It delivers types only. These deps cause different majors of packages in our project.
```
{
    "name": "@types/jest",
    "version": "26.0.23",
    "dependencies": {
        "jest-diff": "^26.0.0",
        "pretty-format": "^26.0.0"
    }
}
```

2) Need types for jest v27.

**Comment by bobaaaaa** [comment]
The jest dependency has its own types. You can delete the `@types/jest` dependency.

**Comment by AgentEnder** [comment]
When I remove the @types/jest dev dep, global functions like describe are no longer being picked up by typescript.

**Comment by bobaaaaa** [comment]
Ah sorry you're right: https://jestjs.io/docs/next/getting-started#using-typescript

**Comment by blazkovicz** [comment]
So, why has the issue been closed?

**Comment by domdomegg** [comment]
In regards to point 2, I've raised a PR to update Jest to version 27: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55013
